### PR TITLE
chore: fix typos, add code of conduct link

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
**What kind of change does this PR introduce?**
typo fixes in test/readme.md, updates code of conduct link in CODE_OF_CONDUCT.md

**Did you add tests for your changes?**
not required

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
nothing

### CODE REVIEW

**Comment by Parikshit-Hooda** [comment]
fixes #10564

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1 +1 @@
-[Code of Conduct](https://js.foundation/community/code-of-conduct)
+[Code of Conduct](https://github.com/openjs-foundation/code-and-learn/blob/master/CODE_OF_CONDUCT.md)
```
Comment (snitin315) [inline CODE_OF_CONDUCT.md]: you should revert this change A PR is already open fixing this one #10566

**Comment by sokra** [comment]
Thanks
